### DISCUSSION TITLE
How to see what's new for each version？

### DISCUSSION THREAD

**Discussion by south-ocean** [jax-ml/jax#23639] [Q&A] [answered]
I now see that the jax has been updated to version 0.4.32, but similar from 0.4.30, I don't know what features were updated in each version and what bugs were fixed, so I don't know if it was necessary for me to update the version of jax. Is there any link to the release note?

**Comment by jakevdp** [comment, marked as answer]
We track changes in the [Change log](https://jax.readthedocs.io/en/latest/changelog.html). We don't list every single change there, but the major ones should be listed, and hopefully it will give you an idea of the main changes from version to version.
